web/src/components/netflow-record/record-field.tsx

```diff
@@ -336,38 +396,34 @@ export const RecordField: React.FC<{
         );
       case ColumnsId.packets:
       case ColumnsId.bytes:
-        let sentText = t('sent');
-        if (c.id === ColumnsId.bytes && flow.fields.IcmpType) {
-          sentText = t('sent reporting {{type}} {{code}}', {
-            type: getType(flow.fields.IcmpType),
-            code: getCode(flow.fields.IcmpType, flow.fields.IcmpCode)
-          });
-        }
-
         //show both sent / dropped counts
         if (Array.isArray(value) && value.length) {
           let droppedText = t('dropped');
-          if (c.id === ColumnsId.bytes && flow.fields.PktDropLatestDropCause) {
-            droppedText = t('dropped by {{reason}}', { reason: flow.fields.PktDropLatestDropCause });
+          let child: JSX.Element | undefined = undefined;
+          if (detailed && c.id === ColumnsId.packets && flow.fields.PktDropLatestDropCause) {
+            droppedText = t('dropped by');
+            child = clickableContent(
+              flow.fields.PktDropLatestDropCause,
+              getDropCauseDescription(flow.fields.PktDropLatestDropCause as DROP_CAUSES_NAMES),
+              'https://elixir.bootlin.com/linux/latest/source/include/net/dropreason-core.h#L88'
```

Comment:
is there a specific reason to use a URL from `elixir.bootlin.com` ? (I don't know about this site)
This one sounds more official / primary source: https://github.com/torvalds/linux/blob/master/include/net/dropreason-core.h ?